src/home/FAQcard/FAQcard.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import './FAQcard.css';
+import logo_left_arrow from './../../assets/images/side_arrow.svg'
+export default function FAQcard(props) {
+
+    return (
+        <section className="faq_card">
```

Comment:
@mtreacy002 @naveen8801 ideally, the faq card should be an `accordion` which should show the answer to the question.

Reply:
hii @Rahulm2310 @mtreacy002, should I add this feature also?

Reply:
to use accordion as card item? it would be great if you could 😉

Reply:
sure @mtreacy002  :-)